src/utils/errors.js

```diff
@@ -0,0 +1,25 @@
+class MissingFieldsError extends Error {
+    message = "Missing fields in request body";
+    code = 400;
+  }
+
+class CantFindIdError extends Error {
+    message = "Could not find record";
+    code = 404;
+  }
+
+class WrongPasswordError extends Error {
+    message = "Wrong password";
```

Comment:
Don't be specific about credential messages, prefer vagueness. "Wrong username or password" regardless of which one is wrong

Reply:
Fixed